frontend/src/metabase/lib/urls/utils.ts

```diff
@@ -17,3 +17,14 @@ function flattenParam([key, value]: [string, unknown]) {
 export function extractQueryParams(query: Record<string, unknown>) {
   return Object.entries(query).map(flattenParam).flat();
 }
+
+export function getEncodedUrlSearchParams(query: Record<string, unknown>) {
+  return new URLSearchParams(
+    extractQueryParams(query).map(([key, value]) => {
+      if (value === null) {
```

Comment:
I think the failing tests are caused by the original `==` being replaced with this more strict `===`

Reply:
Good catch! I missed that. 🙏

Reply:
Fixed in https://github.com/metabase/metabase/pull/36739/commits/1f90b0fca54fcbd90b626caa6b20b9d0c079902d